10/s2i/bin/assemble

```diff
@@ -100,9 +100,7 @@ else
 	if ! mountpoint $NPM_CACHE; then
 		echo "---> Cleaning the npm cache $NPM_CACHE"
 		#As of npm@5 even the 'npm cache clean --force' does not fully remove the cache directory
-		# instead of $NPM_CACHE* use $NPM_CACHE/*.
-		# We do not want to delete .npmrc file.
-		rm -rf "${NPM_CACHE:?}/"
+		rm $NPM_CACHE* -rf
```

Comment:
Please change it back again.
`rm -rf $NPM_CACHE* -rf` removes also ~/.npmrc file which should be present for NPM_REGISTRY settings.

Reply:
Thanks for spotting this. Must have missed this during the rebase. Will fix and force push once more.